artifacts/Bootstrap.ps1

```diff
@@ -11,7 +11,9 @@ param (
     [string]$windowsNode,
     [string]$kubernetesDistribution,
     [string]$uamiClientId,
-    [string]$proxyCredentialsKeyVaultName
+    [string]$proxyCredentialsKeyVaultName,
+    [string]$helmRegistry,
```

Comment:
the SFF baremetal pipeline also using this script, but it does not populate the new parameters, can you test if this change will break the baremetal pipeline?

Reply:
added new parameters to baremetal pipeline in atlas repo